src/us/kbase/workspace/database/provenance/RelationshipType.java

```diff
@@ -155,10 +155,10 @@ public String getPid() {
 	 *                                  relationship type name related to the input string.
 	 */
 	public static RelationshipType getRelationshipType(final String input) {
```

Comment:
```suggestion
	public static RelationshipType getRelationshipType(final String relationshipType) {
```
Make error message match the variable name for easier debugging